albumentations/augmentations/transforms.py

```diff
@@ -1252,7 +1365,44 @@ def get_params_dependent_on_targets(self, params):
         return {"vertices_list": vertices_list}
 
     def get_transform_init_args_names(self):
-        return ("shadow_roi", "num_shadows_lower", "num_shadows_upper", "shadow_dimension")
+        return (
+            "shadow_roi",
+            "num_shadows_lower",
+            "num_shadows_upper",
+            "shadow_dimension",
+        )
+
+
+class RandomToneCurve(ImageOnlyTransform):
+    """Randomly change the relationship between bright and dark areas of the image by manipulating its tone curve.
+
+    Args:
+        scale (float): Amount of random deviation from the original image tone.
+
+    Targets:
+        image
+
+    Image types:
+        uint8
+    """
+
+    def __init__(
```

Comment:
Add add function `get_transform_init_args_names` and describe `scale` as init argument name. It is necessary because all functions must be serializable.
Also add this transform into these tests:
- `tests/test_serialization.py`:
    - `test_augmentations_serialization`
    - `test_augmentations_serialization_with_custom_parameters`
    - `test_additional_targets_for_image_only_serialization`
 - `tests/test_augmentations.py`:
     - `test_image_only_augmentations`
     - `test_augmentations_wont_change_input`
     - `test_augmentations_wont_change_shape_grayscale`
     - `test_augmentations_wont_change_shape_rgb`
     - `test_multichannel_image_augmentations`
     - `test_multichannel_image_augmentations_diff_channels`